DUI3-DX/Sdk/Speckle.Converters.Common.DependencyInjection/ToHost/ConverterWithFallback.cs

```diff
@@ -49,6 +50,10 @@ public object Convert(Base target)
     var displayValue = target.TryGetDisplayValue<Base>();
     if (displayValue != null)
     {
+      if (displayValue is IList && !displayValue.Any())
```

Comment:
This is ideally should be prevented by sender but I don't see why not to report it

Reply:
This is good, there is no guarantee an object will be correctly generated as they can come from anywhere. Sending a wall without a displayValue is _doable_ but not something our connectors would do (except Grasshopper/Dynamo which technically could)